_sass/minimal-mistakes/_base.scss

```diff
@@ -131,62 +132,81 @@ blockquote {
   }
 }
 
+%quote-style {
+  font-weight: bold;
+  font-size: 100px;
+  color: $pyos-teal;
+  line-height: 0;
+}
+
 /* Block quote with bigger quotes*/
 blockquote.highlight-quote {
   border: none;
   font-family: Georgia, "Times New Roman", Times, serif;
-  margin-bottom: 130px;
-  margin-top: 110px;
-  padding: 0 80px 0 30px;
-  max-width: 35rem;
+  margin-top: 6rem;
+  margin-bottom: 4rem;
+  padding: 0 2rem;
+  max-width: $medium;
 
   footer {
-    text-align: left;
     font-style: italic;
     font-weight: 500;
     text-align: right;
   }
 
+  &:after{
+    @extend %quote-style;
+    position: absolute;
+    content: close-quote;
+    right: 4rem;
+    margin-top:2rem;
+  }
+  & > div.end {
+    position: relative;
+  }
+
+  & > div.end:after {
+    @extend %quote-style;
+    content: close-quote;
+    position: absolute;
+    right: 0;
+    margin-top:2.8rem;
+  }
+
   p {
     font-size: 21px;
     position: relative;
     margin-bottom: 10px;
 
-    &:before,
-    &:after {
-      font-weight: bold;
-      font-size: 100px;
-      color: $pyos-teal;
-      line-height: 0;
-    }
-
     &:before {
+      @extend %quote-style;
       position: absolute;
       content: open-quote;
       left: 0;
       top: -20px;
     }
-
-    &:after {
-      content: close-quote;
-      position: absolute;
-      right: 0;
-      bottom: -80px;
-    }
   }
-  // Success class (magenta)
-  &.magenta p {
-    &:before,
-    &:after {
+
+  &.magenta {
+    & > div.end:after,
+    p:before,
+    p:after {
       color: $pyos-magenta;
     }
   }
-  &.purple p {
-    &:before,
-    &:after {
+
+  &.purple {
+    & > div.end:after,
+    p:before,
+    p:after {
       color: $pyos-mediumpurple;
     }
   }
+
+  @include breakpoint('max-width: #$mobile') {
```

Comment:
will go back and look at this more closely. i'm still fighting with mixin include statements